src/component/marker/MarkLineView.ts

```diff
@@ -192,7 +193,7 @@ function markLineFilter(
         }
     }
     return markerHelper.dataFilter(coordSys, item[0])
-        && markerHelper.dataFilter(coordSys, item[1]);
+        || markerHelper.dataFilter(coordSys, item[1]);
```

Comment:
Does this mean that if none of the two points are inside the coordSys, it should always be hidden? Conside the case when two points are outside of the coordSys, but their connecting line has some part inside the coordSys. I think it may make more sense to display the part inside of the coordSys.

Reply:
Good point, I haven't considered that case. It should be displayed

Reply:
Do we need to consider polar system here?

Reply:
> I'm not sure but seems this check is more precise than the previous

Just find out removing this logic will break coord is `Infinity` case which is explain in previous comment
```js
        markLine: {
        data: [{ yAxis: 2 }]
        }
```